### ISSUE TITLE
`update` and non-numeric types

### ISSUE THREAD

**Issue by 12345ieee** [scanmem/scanmem#208]
Steps:
- `option scan_data_type string` (or `bytearray`)
- `" some random string`
- get at least a match
- `update`

Expected:
Unless something changed in the process, everything remains the same.

What I get:

```
43> update 
.error: unsupported scan for current data type.
error: failed to scan target address space.
Please enter current value, or "help" for other commands.
0>
```

Numeric types work well.

**Comment by sriemer** [comment]
This is just not supported. So queuing for `v0.17`.

**Comment by 12345ieee** [comment]
I classed it as a 'bug' because you LOSE the previous matches.
If it just refused to work it'd be fine as 'enhancement'.

**Comment by 12345ieee** [comment]
Fixed in 12345ieee@7afccdc

**Comment by 12345ieee** [comment]
Memo for me, fix this one better.
